.github/workflows/update-rpc.yml

```diff
@@ -0,0 +1,94 @@
+name: Update Submodule
+
+# on:
+#   release:
+#     types: [published]
+
+on: [pull_request]
+
+permissions: write-all
+
+jobs:
+  update-submodule:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Kakarot Repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get Release Tag
+        id: get_release_tag
+        # run: echo "release_tag=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+        run: echo "release_tag=v0.8.2" >> $GITHUB_ENV
```

Comment:
this is just for debugging with `on: [pull_request]` right? otherwise we can use directly the `${{ github.event.release.tag_name }}` ?

Reply:
Yes exactly it was to debug and hardcode a release otherwise I will uncomment the line above to do on release. I have just passed the PR in draft to be sure that we don't merge anything yet.